uiowa/uiowa
# Enable visual indicator on page and person teasers

Resolves: https://github.com/uiowa/uiowa/issues/5057

This PR enables the visual indicator button on teasers for:

- Articles
- Pages
- People

Screenshots:
<img width="800" alt="Screen Shot 2022-04-05 at 8 50 56 AM" src="https://user-images.githubusercontent.com/472923/161769581-06e787db-f2db-4e3e-8581-7f2b6d712df0.png">
<img width="800" alt="Screen Shot 2022-04-05 at 8 52 45 AM" src="https://user-images.githubusercontent.com/472923/161770023-9e63481c-dbb1-4e1e-999a-b5abe8aef6d5.png">

# How to test

1. Check out this branch
2. Pull down the Mental Health website:
`blt ds --site="mentalhealth.uiowa.edu"`
3. Enable visual indicators by visiting the SiteNow Page and SiteNow People configuration pages and checking the "Display arrows linking to pages from lists/teasers." option: `/admin/config/sitenow/sitenow-people`, `/admin/config/sitenow/uiowa-pages`, `/admin/config/sitenow/uiowa-articles`
3. Click on a tag from the "Find Help Now" page (ex: https://mentalhealth.uiowa.ddev.site/tags/faculty-and-staff)
4. See the visual indicators for pages
5. Go to [/people](https://mentalhealth.uiowa.ddev.site/people) and see visual indicators for people
6. Go to [/news](https://mentalhealth.uiowa.ddev.site/news) and see visual indicators for articles
7. Test on featured content blocks
8. Make sure this is only available for a `v3` site.
